Diffuse/Extensions/UITableViewExtensions.swift

```diff
@@ -6,7 +6,9 @@ import UIKit
 
 extension UITableView {
     public func reload(with changes: CollectionChanges,
-                       animation: UITableView.RowAnimation = .automatic,
+                       insertAnimation: UITableView.RowAnimation = .automatic,
+                       reloadAnimation: UITableView.RowAnimation = .automatic,
+                       deleteAnimation: UITableView.RowAnimation = .automatic,
                        section: Int = 0,
```

Comment:
Maybe use a dictionary with enum keys for this?

Reply:
You mean something like this?

```swift
enum TableViewOperation {
	case insert
	case update
	case remove
}

func reload( animations: [TableViewOperation: UITableView.RowAnimation] ) {}
```

Reply:
Yep, something like that :)

Reply:
I'll give it a try :+1:

Reply:
But perhaps in another PR so we can discuss it. I'll merge this and open another one with these changes

Reply:
👍

Reply:
The changes are in PR #6